.github/workflows/ci.yaml

```diff
@@ -30,9 +30,9 @@ jobs:
         # these versions are automatically updated as new releases become 
         # available.
         consul-version:
-        - 1.14.9
-        - 1.15.5
-        - 1.16.1
+        - 1.16.6
+        - 1.17.3
+        - 1.18.0
```

Comment:
```suggestion
        - 1.14.9
        - 1.15.5
        - 1.16.1
```

Reply:
reverting test because CI is blocked and this repo has no clear admin to change it